src/pipeline.ts

```diff
@@ -254,11 +259,9 @@ export async function checkNewCandidateMatch(
 		searchee,
 		candidate.tracker,
 	);
-	await sendResultsNotification(
-		searchee,
-		[[assessment, candidate.tracker, result]],
-		Label.REVERSE_LOOKUP,
-	);
+	sendResultsNotification(searchee, [
```

Comment:
curious as to why `await` was removed here

Reply:
The function is not async.